components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -951,10 +1010,15 @@ private String addLocaleToTemplateTypeResourcePath(String path, String locale) {
      */
     private String buildTemplateRootDirectoryPath(String type, String notificationChannel) {
 
+        return buildTemplateRootDirectoryPath(type, notificationChannel, null);
+    }
+
+    private String buildTemplateRootDirectoryPath(String type, String notificationChannel, String applicationUuid) {
```

Comment:
Change this type to template type -> check in rest api impl as well.

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/223/commits/43afd46f6a19a8fc329ccee7166d75003c866306